marketplaceItems/questionApp/appResources/appData/resources/question_ui.html

```diff
@@ -0,0 +1,17 @@
+<html>
+    <head>
+        <title>Draw</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://fonts.googleapis.com/css?family=Raleway:300,400,600,700" rel="stylesheet">
+        <link rel="stylesheet" href="css/question_ui.css">
+    </head>
+    <body>
+        <h1>Thank you for downloading the question app!</h1>
+        <p class="h4Style">This basic app allows you to signal that you would like to ask a question during meetings or events.</p>
+        <p class="h4Style">Simply toggle the app on to put a question mark above your head. Admins in the domain can then
+            see that you are waiting. The question mark will change color over time to show how long you've been waiting.</p>
+        <p class="h4Style">When an admin is ready for you to ask your question, they will remove your question mark and you will hear a sound. your 
```

Comment:
remove extra carriage return

Reply:
these were to keep a line length of < 128...remove anyway?

Reply:
yeah, i'm okay with long line lengths in HTML files.